devicetypes/smartthings/zigbee-motion-sensor-light-light.src/led-cpx-motion-sensor-child.groovy

```diff
@@ -0,0 +1,53 @@
+/**
+ *  Copyright 2017 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  ITM CPX Motion sensor child
+ *
+ *  Author: SAMSUMG LED
+ *  Date: 2021-12-08
+ */		
+ 						
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus						
+import physicalgraph.zigbee.zcl.DataType						
+						
+metadata {
+	definition (name: "ITM CPX Motion sensor child", namespace: "SAMSUNG LED", author: "SAMSUNG LED", ocfDeviceType: "x.com.st.d.sensor.motion") {
+		capability "Motion Sensor"
+		capability "Configuration"
+		capability "Refresh"
+		capability "Health Check"
```

Comment:
`Health Check` capability, without `ping()` method is pointless. Either remove capability from the child DTH, or include `ping()` method. I'd personally go with 1st option and add a zigbee message to the parent's `ping()` that would bring child back from being offline.